sysutils/beats8/src/opnsense/mvc/app/views/OPNsense/Beats8/filebeat.volt

```diff
@@ -0,0 +1,57 @@
+{#
+
+Copyright (C) 2025 Maxime THIEBAUT
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+1.  Redistributions of source code must retain the above copyright notice,
+this list of conditions and the following disclaimer.
+
+2.  Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+#}
+
+<script>
+    $( document ).ready(function() {
+        mapDataToFormUI({'frm_GeneralSettings':"/api/filebeat/settings/get"}).done(function(data){
+            // place actions to run after load, for example update form styles.
+            updateServiceControlUI('filebeat');
+            $('.selectpicker').selectpicker('refresh');
+        });
+
+        // link save button to API set action
+        $("#saveAct").click(function(){
```

Comment:
I'm ok with this, but simpler option is possible using a `SimpleActionButton()`, example in :

https://github.com/opnsense/core/blob/00921f6b3e6a13cb8c04c6250737c125f276abc2/src/opnsense/mvc/app/views/OPNsense/Dnsmasq/settings.volt#L98-L107

and to add the button:

https://github.com/opnsense/core/blob/00921f6b3e6a13cb8c04c6250737c125f276abc2/src/opnsense/mvc/app/views/OPNsense/Dnsmasq/settings.volt#L205


docs : https://docs.opnsense.org/development/frontend/view_js_helpers.html#simpleactionbutton